core/src/banking_stage.rs

```diff
@@ -4167,15 +4167,15 @@ mod tests {
                 })
                 .collect_vec();
 
+            let non_vote_packet_batches = to_packet_batches(&txs, 10);
             let tpu_packet_batches = to_packet_batches(&tpu_votes, 10);
             let gossip_packet_batches = to_packet_batches(&gossip_votes, 10);
-            let tx_packet_batches = to_packet_batches(&txs, 10);
 
             // Send em all
             [
-                (tpu_packet_batches, tpu_vote_sender.clone()),
```

Comment:
this clone is meaningless. there's no way to influence the receiving side by the timing of `drop()`-ing crossbeam channels. once it's sent, it's guaranteed to be delivered before the receiving channel errors out.